delete_expired_objects/delete_expired_objects_job.py

```diff
@@ -1,32 +1,49 @@
 from time import time
 from gen3 import object, metadata, auth
 
+"""
+Note: This job requires Metadata service to be above version 1.6.4
```

Comment:
1.6.4 does not include this change, please replace with 2022.06 and possibly the semver release you might create